tests/integration/test_vulnerability_detector/test_scan_types/test_partial_scan_type.py

```diff
@@ -0,0 +1,170 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import time
+
+import pytest
+
+import wazuh_testing.vulnerability_detector as vd
+from wazuh_testing.tools import ALERT_LOGS_PATH, LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
+from wazuh_testing.tools.monitoring import FileMonitor
+
+# Marks
+pytestmark = [pytest.mark.server, pytest.mark.tier(level=0)]
+
+local_internal_options = {
+    'wazuh_modules.debug': 2,
+    'monitord.rotate_log': 0
+}
+
+# variables
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+test_feed_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), '..', 'data', 'feeds')
+configurations_path = os.path.join(test_data_path, 'wazuh_partial_scan_type.yaml')
+wazuh_log_monitor = FileMonitor(LOG_FILE_PATH)
+wazuh_alert_monitor = FileMonitor(ALERT_LOGS_PATH)
```

Comment:
I prefer to monitor the `alerts.json` file rather than the plain log. The reason for this change is based on the fact that we can extract the alert using the `FileMonitor` and check the fields in it by import it with the `json` module.

Reply:
Done in: https://github.com/wazuh/wazuh-qa/pull/1428/commits/d0b7c1dfdf179be264e9fde70337ad9223ca8d3f